src/MessageProperty/WireEncryptedStringConversions.cs

```diff
@@ -11,6 +11,17 @@ public static void EncryptValue(this IEncryptionService encryptionService, WireE
             wireEncryptedString.Value = null;
         }
 
```

Comment:
Do we need to make provisions to remove (or obsolete) this code once `NServiceBus.WireEncryptedString` is removed from core?